pkgs/cupertino_http/example/lib/get_books.dart

```diff
@@ -0,0 +1,38 @@
+import 'dart:convert';
+import 'package:http/http.dart';
+
+import 'book.dart';
+
+Future<List<Book>> getBooks(String query) async {
+  final response = await get(
```

Comment:
You don't think that it is valuable so have this comment?

// `get` will use the `Client` configured in main.

Reply:
Yes, good idea, added